sdk/src/sys/debug.rs

```diff
@@ -9,4 +9,8 @@ super::fvm_syscalls! {
 
     /// Logs a message on the node.
     pub fn log(message: *const u8, message_len: u32) -> Result<()>;
+
+    /// TODO: this technically lets anyone store whatever data they want on the node's filesystem, this should *not* be enabled on an actual network until proper gaurds are in place
+    /// Save data as a debug artifact on the node.
```

Comment:
maybe we could put all these things into a special `feature`, and make sure that we can only touch them in tests & benches

Reply:
That would make it difficult to debug from lotus. IMO, an environment variable (artifacts are only saved when set) and some path checks _should_ be sufficient?

Reply:
In the future, we could expand that env var to a set of rules like `FVM_STORE_ARTIFACTS=code:SomeCid=/path/to/dir;addr:f01=/path/to/other-dir`.